sdk/src/main/java/com/google/cloud/dataflow/sdk/io/OffsetBasedSource.java

```diff
@@ -308,6 +319,25 @@ public Double getFractionConsumed() {
     }
 
     @Override
+    public long getParallelismConsumed() {
+      return rangeTracker.getProcessedSplitPoints();
```

Comment:
Would be good to be consistent with getParallelismConsumed: getSplitPointsProcessed?